cmd/dlt/idp/cmd.go

```diff
@@ -58,8 +58,13 @@ func run(_ *cobra.Command, argv []string) {
 	idpName := argv[0]
 
 	clusterKey := r.GetClusterKey()
-
```

Comment:
Super-nit: since you probably have to address Andrea's comment so you might as well undo this change unless it's lint related.

Reply:
removed as sugessted

Reply:
/lgtm